aquadoggo/src/network/service.rs

```diff
@@ -16,111 +16,265 @@ use tokio_stream::StreamExt;
 use crate::bus::{ServiceMessage, ServiceSender};
 use crate::context::Context;
 use crate::manager::{ServiceReadySender, Shutdown};
-use crate::network::behaviour::{Behaviour, BehaviourEvent};
+use crate::network::behaviour::{Event, P2pandaBehaviour};
 use crate::network::config::NODE_NAMESPACE;
-use crate::network::{dialer, identity, peers, swarm, NetworkConfiguration, ShutdownHandler};
+use crate::network::{identity, peers, swarm, NetworkConfiguration, ShutdownHandler};
 
-/// Network service that configures and deploys a libp2p network swarm over QUIC transports.
+static DEFAULT_QUIC_PORT: u16 = 2022;
```

Comment:
I would not have expected the default values to be defined in the service but somewhere close to `NetworkConfiguration`.